deployments/deploy-ssh.sh

```diff
@@ -399,6 +399,7 @@ spec:
             "--spec-dir", $SPEC_DIR,
             "--app-list", "$APP_LIST",
             "--scheduler", "$SCHEDULER",
+            "$MAX_STORAGE_NODES_PER_AZ_ARG",
```

Comment:
use explicit parameter here _--max-storage-nodes-per-az=$MAX_STORAGE_NODES_PER_AZ_ARG_. I have noticed that sometimes if you have empty line in the spec, it could skip all below parameters in the spec

Reply:
done